matchmaking/models.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+
+from users.models import SocialProfile
+from pennychat.models import PennyChat
+
+
+class TopicChannel(models.Model):
+    channel_id = models.CharField(max_length=20, unique=True)
+    slack_team_id = models.CharField(max_length=20)
+    name = models.TextField()
```

Comment:
is `name` the topic associated with the channel or just the actual name of the channel cached so that we don't have to look it up again?

Reply:
Name of the channel so that we don't have to look it up every time.